src/python/examples/simple_grpc_sequence_stream_infer_client.py

```diff
@@ -180,19 +188,22 @@ def async_stream_send(triton_client, values, batch_size, sequence_id,
     for i in range(len(result0_list)):
         seq0_expected = 1 if (i == 0) else values[i - 1]
         seq1_expected = 101 if (i == 0) else values[i - 1] * -1
+        seq2_expected = 21 if (i == 0) else values[i - 1] * -2
         # The dyna_sequence custom backend adds the correlation ID
         # to the last request in a sequence.
+        # Note: It does not support string sequence IDs
```

Comment:
What does the model do for string IDs? You should explain what "does not support" means